app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -206,15 +209,15 @@ export default class AnnotateController {
         this.editHandler.enable();
     }
 
-    createEditableAndDraggableDrawLayer(mapWrapper, geometry, isCreate) {
+    createEditableAndDraggableDrawLayer(mapWrapper, geometry) {
         if (geometry.type === 'Polygon') {
             let coordinates = _.map(geometry.coordinates[0], c => [c[1], c[0]]);
             let polygonLayer = L.polygon(
                 coordinates,
                 {
                     'draggable': true,
-                    'fillColor': isCreate ? '#3388ff' : '#ff4433',
-                    'color': isCreate ? '#3388ff' : '#ff4433',
+                    'fillColor': '#E57373',
```

Comment:
I'd list colors as constants at the top of the file, so they're easier to change all at once if we want to.